src/scope/repositories/sources.ts

```diff
@@ -11,7 +11,7 @@ import GeneralError from '../../error/general-error';
 import logger from '../../logger/logger';
```

Comment:
these changes are from another PR that's already merged to master... Due to overzealous re-basing on my part